src/utils/is-physics.js

```diff
@@ -0,0 +1,3 @@
+const isPhysics = subTopic => subTopic.match(/physics/i);
```

Comment:
not a change for now, but for this util and the integrated math one, instead of a regex match could you could do something like this?
`Object.keys(PHYSICS_MAPPING).includes(subTopic)`